src/cloud-api-adaptor/Makefile

```diff
@@ -22,6 +22,7 @@ SOURCES     := $(shell find $(SOURCEDIRS) -name '*.go' -print)
 TEST_E2E_TIMEOUT ?= 60m
 
 RESOURCE_CTRL ?= true
+YQ_CHECKSUM_${ARCH} ?= $(YQ_CHECKSUM)
```

Comment:
would it make sense to drop YQ_CHECKSUM in favor of YQ_CHECKSUM_x86_64 and YQ_CHECKSUM_s390x?

Reply:
Hi @snir911, Line 10 substitutes 'x86_64' with 'amd64' in the 'ARCH' value if it's not provided, potentially introducing inconsistency? Could you please suggest 

 ```ARCH ?= $(subst x86_64,amd64,$(shell uname -m))```

Reply:
That's odd, isn't the Docker file always expects x86_64?  we are passing it amd64